server/channels/store/sqlstore/user_store.go

```diff
@@ -2268,6 +2268,47 @@ func (us SqlUserStore) RefreshPostStatsForUsers() error {
 	return nil
 }
 
+func applyUserReportFilter(query sq.SelectBuilder, filter *model.UserReportOptions, isPostgres bool) sq.SelectBuilder {
+	query = applyRoleFilter(query, filter.Role, isPostgres)
+	if filter.HasNoTeam {
+		query = query.Where(sq.Expr("u.Id NOT IN (SELECT UserId FROM TeamMembers WHERE DeleteAt = 0)"))
+	} else if filter.Team != "" {
+		query = query.Join("TeamMembers tm ON (tm.UserId = u.Id AND tm.DeleteAt = 0)").
+			Where(sq.Eq{"tm.TeamId": filter.Team})
+	}
+	if filter.HideActive {
+		query = query.Where(sq.Gt{"u.DeleteAt": 0})
+	}
+	if filter.HideInactive {
+		query = query.Where(sq.Eq{"u.DeleteAt": 0})
+	}
+
+	if strings.TrimSpace(filter.SearchTerm) != "" {
+		query = generateSearchQuery(query, strings.Fields(sanitizeSearchTerm(filter.SearchTerm, "*")), UserSearchTypeAll, isPostgres)
```

Comment:
Is this enough to protect against SQL injection? I want to make sure I'm not exposing anything here (even though it's only accessible via an admin endpoint)

Reply:
Yup, as long we we don't use user input to construct a SQL string, it will be fine, the driver will send the query and params separately, and the database will ensure to not _execute_ the query params.